vyper/evm/opcodes.py

```diff
@@ -26,6 +26,7 @@
     "berlin": 3,
     "paris": 4,
     "shanghai": 5,
```

Comment:
What is the proper way to handle this here? Should I skip shanghai completely? I was worried it might cause problems somewhere else. For example the --evm-version parameter will not accept it anymore (not that it changes anything). If other forks are missing I guess it's totally ok to remove it.

Reply:
@charles-cooper I think mentioned it's fine if we don't have all forks defined here.